core/src/main/java/org/apache/calcite/runtime/SqlFunctions.java

```diff
@@ -816,6 +816,33 @@ public static String charFromUtf8(int n) {
     return String.valueOf(Character.toChars(n));
   }
 
+  /**
+   * SQL CODE_POINTS_TO_BYTES function.
+   */
+  public static @Nullable ByteString codePointsToBytes(@Nullable List codePoints) {
+    if (codePoints == null) {
```

Comment:
I think you can and should remove the `@Nullable` from `codePoints`. The function should not be called if `codePoints` is null.